docker-compose.regtest.yml

```
@@ -1,20 +1,60 @@
     environment:
-      - ARKD_WALLET_ESPLORA_URL=http://chopsticks:3000
-      - ARKD_WALLET_NEUTRINO_PEER=bitcoin:18444
+      - ARKD_WALLET_LOG_LEVEL=5
+      - ARKD_WALLET_NBXPLORER_URL=http://nbxplorer:32838
       - ARKD_WALLET_DATADIR=./data/regtest
       - ARKD_WALLET_NETWORK=regtest
     volumes:
       - type: tmpfs
         target: /app/data
-
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)